samples/client/petstore/java/okhttp-gson-dynamicOperations/pom.xml

```diff
@@ -291,7 +291,7 @@
         <dependency>
             <groupId>org.apache.oltu.oauth2</groupId>
             <artifactId>org.apache.oltu.oauth2.client</artifactId>
-            <version>1.0.1</version>
+            <version>1.0.2</version>
```

Comment:
thanks for the PR. we were using 1.0.2 before and there was issue reported and we rolled it back to 1.0.1.

is it correct to say that you've tested the output and works in your use case?

I'll try to loop in the one who rolled back the version to review this PR as well.

Reply:
Thank you for the reply. Sorry for the late response, I lost track of GitHub notifications.

Regarding whether it works - I would say yes - this template change and the pom.xml change is already in production in the project where I am. I can't say we've tested it extensively for all possible OAuth2 flows, but it does work for us.